saas/renewals.py

```diff
@@ -173,11 +173,24 @@ def extend_subscriptions(at_time=None, dry_run=False):
                             '- %s' % subscription.organization.printable_name)
                     else:
                         descr_suffix = None
+                    items = [Transaction.objects.new_subscription_order(
```

Comment:
Kind of the code that should be in `record_use_charge`. Note that the over-quota use charges need to be recognized in `renewals.py:_recognize_subscription_income`. Because use charges are billed after usage, the revenue recognition should be done right at the period end (no backlog).